vipps-login-api-faq.md

```diff
@@ -36,6 +36,7 @@ Document version 2.3.1.
 - [Can we have multiple URIs as landing pages?](#can-we-have-multiple-uris-as-landing-pages)
 - [Can I use a custom URL scheme for the `redirect_url`?](#can-i-use-a-custom-url-scheme-for-the-redirect_url)
 - [Can we change the name that appears in customer's Vipps app under `Login and Access`?](#can-we-change-the-name-that-appears-in-customers-vipps-app-under-login-and-access)
+- [How can I logout a user?](#how-can-i-logout-a-user)
```

Comment:
"log out" in two words?

Reply:
"log out" is to take an action, so maybe?